components/ChatPage/ExcelUpload.tsx

```diff
@@ -36,28 +34,25 @@ function ExcelUpload({ convUid, chatMode, fileName, onComplete, ...props }: Prop
     try {
       const formData = new FormData();
       formData.append('doc_file', fileList[0] as any);
-      const { success, err_msg } = await axios.post<unknown, IResponseModal>(
-        `/api/v1/chat/mode/params/file/load?conv_uid=${convUid}&chat_mode=${chatMode}`,
-        formData,
-        {
-          /** timeout 1h */
-          timeout: 1000 * 60 * 60,
-          headers: {
-            'Content-Type': 'multipart/form-data',
+      const [err] = await apiInterceptors(
```

Comment:
The front and back ends should be named as consistently as possible. If not, the frontend should be consistent, and `scene` is recommended.